buildSrc/src/main/java/Dependencies.kt

```diff
@@ -22,7 +22,7 @@ private object Versions {
     const val androidx_safeargs = "1.0.0-beta01"
 
     const val appservices_gradle_plugin = "0.3.1"
-    const val mozilla_android_components = "0.46.0-SNAPSHOT"
+    const val mozilla_android_components = "0.47.0"
```

Comment:
Btw. if you append "-SNAPSHOT" for `mavenLocal` builds then you shouldn't need to clear the cache for **every** build.

Reply:
Thanks for the heads up!